ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+
+:meta-keywords: server definition, create server, drop server, alter server, rename server
+:meta-description: Define servers in CUBRID database using create server, alter server, drop server and rename server statements.
+
+
+********************
+서버(SERVER) 정의문

```

Comment:
It seems to me that (SERVER) is rather redundant.

Reply:
I agree.